src/data-access/repositories/test/mongo-test-database.ts

```diff
@@ -11,12 +11,12 @@ export class MongoTestDatabase {
     jest.setTimeout(15000)
   }
 
-  public async setupDatabase(): Promise<void> {
+  public async setupDatabaseConnection(): Promise<void> {
     this.mongoServer = await MongoMemoryServer.create()
     this.client = await MongoClient.connect(this.mongoServer.getUri())
   }
 
-  public async teardownDatabase(): Promise<void> {
+  public async teardownDatabaseConnection(): Promise<void> {
```

Comment:
If I were to call this method, I would expect that only the database connection is closed – not the entire database server.

Reply:
Renamed similarly as the setup method.